server/src/services/folder_apps.service.ts

```diff
@@ -11,6 +11,14 @@ export class FolderAppsService {
   ) {}
 
   async create(folderId: string, appId: string): Promise<FolderApp> {
+    const oldFolderApp = await this.folderAppsRepository.findOne({
+      where: { appId, folderId },
+    });
+
+    if (oldFolderApp != null) {
```

Comment:
`if (!oldFolderApp)`
will be good, we can capture undefined value also

Reply:
I have added the check. Please check.